README.md

```diff
@@ -182,6 +182,27 @@ paymentWatcher.start(new WatcherListener<PaymentInfo>() {
 ```
 When you longer want to watch payment, stop the notifications and unregister the listener using `PaymentWatcher.stop()` method.
 
+### Watching Account Creation
+When the account is first created by another account it can be observed.
```

Comment:
maybe move it inside the onboarding section? as its purpose is for onboarding process

Reply:
not sure - It is better to place it near payment watcher I will add internal link from on boarding to account creation watcher

Reply:
excellent